org.groestlcoin.groestlcoin-qt.json

```diff
@@ -0,0 +1,80 @@
+{
+  "id": "org.groestlcoin.groestlcoin-qt",
+  "runtime": "org.freedesktop.Platform",
+  "sdk": "org.freedesktop.Sdk",
+  "runtime-version": "19.08",
+  "command": "groestlcoin-qt",
+  "finish-args": [
+    /* Wayland and Xorg support */
+    "--socket=x11",
+    "--socket=wayland",
+    /* Sound */
+    "--socket=pulseaudio",
+    /* OpenGL access. */
+    "--device=dri",
+    /* IPC access. */
+    "--share=ipc",
+    /* Network acess. */
+    "--share=network"
+  ],
+  "modules": [
+    {
+      "name": "groestlcoin-core",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoind",
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoin-qt",
+        "install -m 0744 -D -t /app/bin/internal bin/groestlcoin-cli"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "only-arches": ["x86_64"],
+          "url": "https://github.com/Groestlcoin/groestlcoin/releases/download/v2.18.2/groestlcoin-2.18.2-x86_64-linux-gnu.tar.gz",
+          "sha256": "9ee26e1cd7967d0dc88670dbbdb99f95236ebc218f75977efb23f03ad8b74250"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["arm"],
+          "url": "https://github.com/Groestlcoin/groestlcoin/releases/download/v2.18.2/groestlcoin-2.18.2-arm-linux-gnueabihf.tar.gz",
+          "sha256": "93c093b7684623c1cdf864cfacdec653ac6808d01f48ec9d1ffe26479623c6f5"
+        },
+        {
+          "type": "archive",
+          "only-arches": ["aarch64"],
+          "url": "https://github.com/Groestlcoin/groestlcoin/releases/download/v2.18.2/groestlcoin-2.18.2-aarch64-linux-gnu.tar.gz",
+          "sha256": "99f7a11f9f59c09f44b64f5631b73b7e98cb27a8fc35ba67c85310001b644e43"
+        }
+      ]
+    },
+    {
+      "name": "misc",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dp -m 644 org.groestlcoin.groestlcoin-qt.desktop /app/share/applications/org.groestlcoin.groestlcoin-qt.desktop",
+        "install -Dp -m 644 groestlcoin.svg /app/share/icons/hicolor/scalable/apps/org.groestlcoin.groestlcoin-qt.svg",
+        "install -Dp -m 644 org.groestlcoin.groestlcoin-qt.appdata.xml /app/share/metainfo/org.groestlcoin.groestlcoin-qt.appdata.xml",
+        "bash install_wrappers.sh"
+      ],
+      "sources": [
+        {
+          "type": "file",
+          "path": "org.groestlcoin.groestlcoin-qt.desktop"
+        },
+        {
+          "type": "file",
+          "path": "org.groestlcoin.groestlcoin-qt.appdata.xml"
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html